docs/_writing-tests/file-names.md

```diff
@@ -54,6 +54,11 @@ themselves precede any test type flag, but are otherwise unordered.
  : (js files only) Indicates that the file generates tests in which it
     is [run in multiple scopes][multi-global-tests].
 
+`.optional`
+ : Indicates that a test makes assertions about optional behavior in a
+   specification, typically marked by the [RFC 2119] "MAY" or "OPTIONAL"
```

Comment:
What about "should"?

Reply:
I think in practice we're just adding tests for such requirements, or at least I've never seen a discussion about whether it's okay to add such tests.

Reply:
Hmm, for some reason I thought that's what spawned this discussion. Maybe call that out explicitly here, to reduce confusion?

Reply:
I've added a bit about should and must.

Reply:
That was my understanding of @jgraham's view (that adding such tests often has little benefit).